### DISCUSSION TITLE
Issues in EDC controlplane

### DISCUSSION THREAD

**Discussion by ds-mwesener** [eclipse-tractusx/tractusx-edc#610] [General]
Dear Team,

is this a known issue?

## Controlplane LOG

DEBUG 2023-07-13T05:34:06.128467308 DSP: Incoming TransferCompletionMessage for transfer process: 89e0cce3-2ed8-4504-8dcc-f64c8b055a20
89
SEVERE 2023-07-13T05:34:06.136936682 JerseyExtension: Unexpected exception caught
88
org.eclipse.edc.spi.EdcException: Current runtime does not hold the lease for Object (id [fb705610-f2d6-44a3-8c8d-7723205b07f6]), cannot break lease!

## Description
My edc installation on edc 0.4.1 is working correctly but throws a lot of those exceptions and therefore takes very long to complete the process.

Thanks in advance for your help.

**Comment by wolf4ood** [comment]
Hi @ds-mwesener 

We released newer version of tranctusx-edc. The latest one is 0.5.0  can you check if it's still happening with that release?

Thanks

**Comment by ds-mwesener** [comment]
HI @wolf4ood unfortunately it is still there: EDC 0.5.0

![image](https://github.com/eclipse-tractusx/tractusx-edc/assets/124587888/b419083d-562c-4266-b425-6c75e3d3f7e8)

Could you please support. We trying to resolve some elements from the registry which takes basically a big amount of time (2-3 hours) for 6 elements due to several retries of the EDC.

At the end it will be successful, though.

Thanks in advance for your help.

**Comment by wolf4ood** [comment]
Hi @ds-mwesener 

which kind of operation takes long time for processing? transfer process?

Thanks

**Reply by ds-mwesener** [reply]
Hi @wolf4ood 
Yes it is the transfer process:

![image](https://github.com/eclipse-tractusx/tractusx-edc/assets/124587888/545919fa-defd-42ba-9a8d-efae5f55120e)

**Comment by ds-mwesener** [comment]
Hello, this issue could be resolved. 
Root cause: We had two instances of edc connected to one instance of the edc postgres database.
